sdk/remoterendering/azure-mixedreality-remoterendering/pyproject.toml

```diff
@@ -2,3 +2,4 @@
 pyright = false
 type_check_samples = false
 verifytypes = false
+pylint = false
```

Comment:
Any particular reason to disable pylint ?

Reply:
I followed the steps that were outlined in the doc linked by the pylint error. The result: pylint crashes now on the repo. It looks like we are hitting [pylint-dev/pylint#5935](https://github.com/pylint-dev/pylint/issues/5935)
```
"TypeError: argument of type 'Const' is not iterable"
```
As we can't upgrade pylint the only way to fix this is to disable it entirely from what I can tell. Any suggestions how to fix this?

Reply:
Seems to work again now that the code has changed. So pylint is enabled again.